### DISCUSSION TITLE
Cant connect to any Outline server

### DISCUSSION THREAD

**Discussion by MADNESSZ1RA** [OutlineFoundation/outline-sdk#225] [Q&A]
i got my own outline server. i got the same access key for my phone and desktop, but on the desktop, outline shows an error "sorry, we could not configure your system to connect to Outline. Please try running the installer again. If you still cannot install Outline, please get in touch with us and let us know that the TAP device failed to install with error code 1". I think that it is a network error, because outline didnt add a new network connector in control panel. how can i fix this?
![photo_2024-04-23_20-59-40](https://github.com/Jigsaw-Code/outline-sdk/assets/92206761/1642f370-dfc9-4b0d-b090-99dfec5f7187)
![image](https://github.com/Jigsaw-Code/outline-sdk/assets/92206761/d9bdc61d-9e0f-43c8-b9b2-53cdfcadc269)

**Comment by amircybersec** [comment]
@MADNESSZ1RA I am not the right person to address/fix this issue but I think it would help the team if you could provide some more information such as your operation system version, as well as the Outline client version you tried to install. 

BTW, did you try to install with admin privileges?

**Comment by fortuna** [comment]
Please file issues for the Outline Client in its repository at https://github.com/Jigsaw-Code/outline-apps/issues
Or contact support@getoutline.org
